fits-loader-docker/Dockerfile

```diff
@@ -4,7 +4,7 @@ COPY . ./
 RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags "-X main.Prefix=${i}/${VERSION} -extldflags -static" -o fits-loader .
 
 FROM quay.io/geonet/python:alpine3.11
-RUN apk add --no-cache bash jq
+RUN apk add --no-cache bash jq ca-certificates tzdata
```

Comment:
can we build ca-certificates from another layer/stage? and copy over to this layer?

whoat tzdata library doing, any timing function calling?